templates/jobs.yaml

```diff
@@ -342,6 +341,8 @@ spec:
           - |
             /bin/cp -aR /opt/stackstorm/packs/. /opt/stackstorm/packs-shared &&
             /bin/cp -aR /opt/stackstorm/virtualenvs/. /opt/stackstorm/virtualenvs-shared
+      {{ end }}
+      {{- if (.Values.st2.packs.image.repository) or (.Values.st2.packs.nfs.server) }}
```

Comment:
Default system packs should be always installed and registered, irrespective of the custom packs.

Reply:
https://github.com/StackStorm/stackstorm-ha/blob/85c21ed9e925b6fabcbcdef6eae3e5ee04cd8b34/templates/jobs.yaml#L327-L358

On master it seems that if `st2.packs.image.repository` is not set the whole `initContainers` section of the job is omitted, I tried to mirror this behavior, I might have made a mistake though.

Reply:
That's right, confirming. I misread the diff.

The default pack content is already present in st2actionrunner image and is registered if no custom packs are provided.